tests/acceptance/external_studies/openprompt-long-covid-vaccines/analysis/dataset_definition_longcovid_prevaccine.py

```diff
@@ -115,16 +115,16 @@
 # covid tests ------------------------------------------------------------
 dataset.first_test_post_lc = sgss_covid_all_tests \
     .where(sgss_covid_all_tests.is_positive) \
-    .where(sgss_covid_all_tests.specimen_taken_date.is_between(dataset.pt_start_date, dataset.pt_end_date)) \
```

Comment:
This is a correct translation of the semantics of the original query, but I wonder if it's what the author actually intended? I think we'll need to confirm with them in each case.

Reply:
I checked with the study author, they did intend to use the `is_between()` as it was implemented, i.e., excluding the start and end date